lab1/src/task2/Student.java

```diff
@@ -0,0 +1,30 @@
+package task2;
+
+public class Student {
+    private String name;
+    private double grade;
+
+    public Student (String _name, double _grade) {
+        name = _name;
```

Comment:
Again avoid using identifiers starting with _ . You could have put for each field this.fieldName = fieldName